code/datums/components/crafting/crafting.dm

```diff
@@ -458,28 +458,40 @@
 
 	return data
 
+/datum/component/personal_crafting/proc/make_action(datum/crafting_recipe/recipe, mob/user)
+	var/atom/movable/result = construct_item(user, recipe)
+	if(!istext(result)) //We made an item and didn't get a fail message
```

Comment:
Created a bug in making this an early return (also may as well update the comment)
```suggestion
	if(istext(result)) //We failed to make an item and got a fail message
```